src/main/java/seedu/manager/Main.java

```diff
@@ -5,17 +5,24 @@
 import seedu.manager.exception.InvalidCommandException;
 import seedu.manager.parser.Parser;
 import seedu.manager.ui.Ui;
+import seedu.manager.storage.Storage;
+
+import java.io.IOException;
 
 public class Main {
     private static final Ui ui = new Ui();
     private static EventList events = new EventList();
+    private static final String filePath = "events.txt";
```

Comment:
This static variable seems to me to be a constant, so I think it should be named following the convention for constants?